filesystem/azure.go

```diff
@@ -212,7 +212,7 @@ func (f *AzureFile) Write(b []byte) (int, error) {
 		return 0, err
 	}
 	if err == io.EOF {
-		return 0, fmt.Errorf("Need blob content")
+		return 0, fmt.Errorf("Need write content")
```

Comment:
I understand that part (you want a separate error for no user data input). That is why I asked you to check it before you start reading. When you starting reading, then it would be just a normal case. Right?